docs/source/scientific_reference/1_time_series.rst

```diff
@@ -0,0 +1,165 @@
+.. _sciref_series:
+
+Scientific reference: time series
+=================================
+
+``sktime``'s conceptual model is based on a number of "time series" related mathematical
+objects:
+
+* "time series", observations of a single process made over time.
+Example: average temperature in London, on a given day.
+* "time series panels", collections of multiple time series, independently observed.
+Example: fever curves of 10 specific hospital patients, for the time in the hospital.
+* "hierarchical time series", collections of multiple time series where individual
+instances are observed at instances defined by variables defining the hierarchy.
+Example: gummy bear sales at 100 specific stores, in 10 federal states.
+
+This document provides a formal reference for the above.
+The objects are defined in terms of:
+
+* abstract data types (ADT) - defines domain, type and common operations
+* scientific types (scitype) - ADT plus statistical assumptions
+
+
+Primitive values - the `Primitives` scitype
+-------------------------------------------
+
+The following sets/domains are called basic primitive domains:
+
+* the real numbers, i.e., the set :math:`\mathbb{R}`
+* the set of boolean truth value, i.e., the two-element set containing `True` or `False`
+* the set of all finite length strings of ASCII characters
+* any other finite set :math:`\mathcal{C}`, called "category set" in this context
+
+A set/domain is called primitive domain if it is a basic primitive domain, or
+finite Cartesian products of basic primitive domains.
+
+An object is said to be of primitive type if it is an element of a primitive domain.
+
+We use the scitype `Primitives` to type objects of primitive type.
+
+
+Time Series - the `Series` scitype
+----------------------------------
+
+An indexed series, with value domain :math:`\mathcal{X}`, index domain :math:`\mathcal{T}`,
+of length :math:`T`, is a collection of index points :math:`t_1, \dots, t_T` and observations
+:math:`x_1, \dots x_T`. We say (and interpret) that :math:`x_i` is the value observed at index
+:math:`t_i`.
```

Comment:
the challenge is that "being a time series" is not a property of the object, it is a matter of typing (attaching interpretation). Since a real index does not necessarily have to be "time". Could be "distance" or "wavelength", as in common "time" series classification applications.

Reply:
i am not sure how your comment is a response to my comment...?

Reply:
It's an answer in that option 1 is not well-defined given the framework.

> and then define a time series as a particular kind of indexed series"

Not an option because: "being a time series" is not a property of an indexed series. At least not how it is defined currently.
Therefore, a time series cannot be a "particular kind of" indexed series, because that would have to be defined by a property.

Why is it not a property? Say, you define a time series to be of real domain. As stated above, the reals could also be "wavelength" or "distance".

Reply:
I do not fully understand the nuance here.
1. How about this wording instead: 'we first define indexed series and then time series'. Or, have a contents section at the top of the page which says that you define indexed series and time series in the section. Or some other way to indicate that we are defining two different things in this section.
2. Is the following statement true: 'all time series are indexed series'. If no - then I am confused. If yes, then this is the sense in which I mean that time series are a 'particular kind' of indexed series.